src/presence_analyzer/tests.py

```diff
@@ -2,12 +2,14 @@
 """
 Presence analyzer unit tests.
 """
-import os.path
 import json
+import os.path
```

Comment:
`import datetime`
`import json`
`import os.path`
`import unittest`
Alphabetically, like this? Is it okey, to use auto-sorting sublime plugin?